### PR TITLE
chore(sdk): queue tests should poll the approx_size method till right value is retrieved

### PR DESCRIPTION
**Pull request by revitalbarletz**
*By submitting this pull request, I confirm that my contribution is made under the terms of the [Monada Contribution License](https://docs.winglang.io/terms-and-policies/contribution-license.html)*.

Queue.approxSize provides an estimation of the queue's size. Consequently, when adding messages to the queue, there may be a brief delay of a few seconds until the actual value is retrieved.

### CODE REVIEW

File: examples/tests/sdk_tests/queue/purge.w [hunk 1]
```diff
@@ -1,12 +1,39 @@
 bring cloud;
 
 let q = new cloud.Queue();
+class TestHelper {
+  init(){}
+  extern "../external/sleep.js" inflight sleep(milli: num);
```
[the remark this answers is not in the document]
Comment (revitalbarletz) [reply examples/tests/sdk_tests/queue/purge.w:6]: The class or the method?

[the remark this answers is not in the document]
Comment (revitalbarletz) [reply examples/tests/sdk_tests/queue/purge.w:6]: @tsuf239 can you review my questions?

[the remark this answers is not in the document]
Comment (tsuf239) [reply examples/tests/sdk_tests/queue/purge.w:6]: the method :)

**Comment by tsuf239** [comment]
Can you please run `npx nx run test hangar` and then `npm run build`? it seems like the snapshots aren't updated...

**Pull request merged**